ui/angular.json

```diff
@@ -1,5 +1,8 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```

Comment:
what is this

Reply:
it add by it self
delete it?
--------------------------------------------------
for triple auth folder because in task need two folder and command that create module create another one
make it two folder just?

Reply:
module should be in the root of it`s directory

Reply:
can you check